docs/excel/custom-functions-dialog.md

```diff
@@ -1,31 +1,31 @@
 ---
 ms.date: 05/03/2019
-description: Create dialogs through custom functions in Excel using JavaScript.
-title: Custom functions dialogs (preview)
+description: Create a dialog box through custom functions in Excel using JavaScript.
```

Comment:
I got feedback to take out box in an email from Reeza but I'm neutral either way